Develop

Hladkyi03 opened the pull request:
https://Hladkyi03.github.io/Museum_2/
Hi! After adding eng language to site scroll-top btn are visible on the start of page, but after scrolling 1px it`s works correctly. Can u tell me what i need to change to fix it. Everything else is working. Buttons register in closest events section don`t do anything, the checklist does not say where they should lead

TarasHoliuk commented:
1. Make the input color transparent to match the design:
![image](https://github.com/mate-academy/Museum_2/assets/86923089/a47232c2-3fa7-400e-a6b6-26da341f3449)

2. Make it links to corresponding social media pages (to the main page)
![image](https://github.com/mate-academy/Museum_2/assets/86923089/1b4f9cc3-abfe-4070-8c50-780f56078cda)

3. Make the logo a link to home (`#`), don't forget about the logo in the menu:
![image](https://github.com/mate-academy/Museum_2/assets/86923089/2789e5fe-4f49-40e3-bbe3-c225ba50132d)

P.S. Didn't get your question, looks like "scroll-top" button works as expected and doesn't appear at the start of the page. About "Register" button: let's make it scroll to the section with email input.

TarasHoliuk reviewed (changes requested):
Looks amazing :fire: 
I've added a few comments to improve it a bit more)

Hladkyi03 commented:
Hi again, i fixed button already, added changes you requested, also added animations when scrolling. Trying it first time, will be grateful to receive feedback about it

SanyaBratashchuk reviewed (changes requested):
Nice job!
1. Update year 
![image](https://github.com/mate-academy/Museum_2/assets/97547824/d5fbdb81-8a52-41a0-8852-e848bbc15669)

SerhiiKirik reviewed (approved):
LGTM 🔥
